### ISSUE TITLE
Error listen tcp 0.0.0.0:5432: bind: address already in use

### ISSUE THREAD

**Issue by sigfrid** [rancher/rancher#9860]
When I try to  a PostgreSQL container I get the error below.

```
postgres (Expected state running but got error: Error response from daemon: driver failed programming external connectivity on endpoint r-x-postgres-1-03f31043 (3b8469d29f89fda0da9a6cc9497f07760e87c1639f5dd8dccc900fcdc16ddd9d): Error starting userland proxy: listen tcp 0.0.0.0:5432: bind: address already in use).
```

The host is not running anything else (besides Rancher system containers).

```
rancher/scheduler:v0.8.2
rancher/agent:v1.2.6
rancher/dns:v0.15.1
rancher/network-manager:v0.7.4
rancher/metadata:v0.9.2
rancher/healthcheck:v0.3.1
rancher/net:v0.11.2
rancher/net:holder
rancher/net:v0.11.2
```

On the host, I have tried to see if anything is listening on port 5432 but it seems to me that is not the case.

ps -fp 5432
UID PID PPID C STIME TTY TIME CMD

For a complete picture, here my compose file

```
postgres:
image: apply/postgres
volumes:
- /usr/postgres/data:/var/lib/postgresql/data
- /usr/postgres/dump:/usr/postgres-dump
environment:
POSTGRES_PASSWORD: nm4x2ft2
DATABASE_URL: postgres://postgres:nm4x2ft2@localhost:5432/apply
ports:
- 5432:5432
network_mode: “bridge”
```

---
| Useful | Info |
| :-- | :-- |
|Versions|Rancher `v1.6.8` Cattle: `v0.183.16` UI: `v1.6.23` |
|Access|`localauth` `admin`|
|Orchestration|`Cattle`|
|Route|`stacks.new`|

**Comment by anzersy** [comment]
Could you please paste the output of command(ss -nlp |grep 5432) here? Actually the parameters of `ps -p` command should be the PID rather than port number.

**Comment by superseb** [comment]
How often does this happen and what OS and Docker version are you using?

**Comment by superseb** [comment]
Let us know if we can assist on this.

**Comment by waqarulshams** [comment]
Sudo ss -nlp |grep 5432
Result:
u_str  LISTEN     0      128    /var/run/postgresql/.s.PGSQL.5432 26230                 * 0                  
tcp    LISTEN     0      128    127.0.0.1:5432                  *:*  

I also have same problem

**Comment by zlc000190** [comment]
i also have same problem `docker: Error response from daemon: Ports are not available:listen tcp 0.0.0.0:5432: bind: address already in use`
